bitnami/minio/templates/statefulset.yaml

```diff
@@ -173,6 +177,18 @@ spec:
           secret:
             secretName: {{ include "minio.secretName" . }}
         {{- end }}
+        {{- if .Values.tls }}
+        - name: minio-certs
+          secret:
+            secretName: {{ .Values.tls.secretName }}
```

Comment:
I would add a verification in the NOTES.txt to check that if TLS is enabled the `secretName` is provided, if not this will fail with a template error.

Reply:
Another approach could be to generate certificates if TLS is enabled but a secret was not provided as we do in the Harbor chart

Reply:
> 
> 
> Another approach could be to generate certificates if TLS is enabled but a secret was not provided as we do in the Harbor chart

Yes could be a good idea.